editor/scene_tree_dock.cpp

```diff
@@ -1587,6 +1587,12 @@ void SceneTreeDock::_selection_changed() {
 	if (selection_size > 1) {
 		//automatically turn on multi-edit
 		_tool_selected(TOOL_MULTI_EDIT);
+	} else {
```

Comment:
Could maybe save one indentation level with `else if`, no?

Reply:
Correct, fixed that.